packages/network-of-terms-query/src/terms.ts

```diff
@@ -55,6 +57,7 @@ export class TermsTransformer {
     ['http://www.w3.org/2004/02/skos/core#related', 'relatedTerms'],
     ['http://www.w3.org/2008/05/skos#related', 'relatedTerms'],
     ['http://www.w3.org/2004/02/skos/core#inScheme', 'inScheme'],
+    ['http://jena.apache.org/text#score', 'score'],
```

Comment:
Of course, this is not the namespace that we want to use for storing the score in queries. It should be decoupled from any particular SPARQL store.

@EnnoMeijers Do you have ideas for a generic RDF vocabulary concerning search results and their relevance scores?

Reply:
Hmm, good question! Maybe use http://purl.org/voc/vrank, it seems to be around for quite some time, the vrank:rankValue property could be appropriate? Vrank has also properties to describe the ranking method in more detail.

Reply:
Good suggestion, thanks!